api/README.md

```diff
@@ -234,17 +234,17 @@ How to Run the Full System (Two Services)
 This powers the actual meta-agent chat orchestration (TaskInterpreter, AgentYAMLBuilder, WorkflowYAMLBuilder).
 
 ```
-maestro serve ./demos/workflows/meta-agents-v2/SELECTED_META_FILE/agents.yaml ./demos/workflows/meta-agents-v2/SELECTED_META_FILE/workflow.yaml
+maestro serve ./workflows/meta-agents-v2/SELECTED_META_FILE/agents.yaml ./workflows/meta-agents-v2/SELECTED_META_FILE/workflow.yaml
```

Comment:
May be worth the added detail to change to  `./../maestro-demos/workflows/meta-agents-v2/SELECTED_META_FILE/agents.yam` as the path on this and the two below.

Reply:
Let's do this after @AngeloDanducci